lib/tokenlist.cpp

```diff
@@ -317,8 +323,37 @@ void TokenList::insertTokens(Token *dest, const Token *src, nonneg int n)
 
 bool TokenList::createTokens(std::istream &code, const std::string& file0)
 {
+#ifdef ASSERT_LANG
```

Comment:
This is temporary and the intention is to simply disable the define in source. I can already provide a PR so it will not be missed. In case I am able to make the appropriate improvements the asserts will one by one go away. In case I am not they will not be enabled at all. Not pretty but it quite local and does not spill into other code.

I do not want an I do not want an `InternalError` for this as this is nothing the user should ever see. We will also not be able to generate the appropriate information the user needs to provide so we can figure out the location in case the source cannot be provided. Also this would require modifications in daca so I get the desired information. That would cause a ripple effect I want to avoid.

Reply:
at least I'd suggest that we avoid ifdefs over and over.. for instance:

```
#ifndef N_ASSERT_LANG
#include <cassert>
#define ASSERT_LANG(cond)  assert(cond)
#endif
```

Reply:
Will do.